protected/humhub/modules/content/components/ContentContainerDefaultPermissionManager.php

```diff
@@ -28,9 +28,21 @@ class ContentContainerDefaultPermissionManager extends PermissionManager
      */
     protected function getModulePermissions(\yii\base\Module $module)
     {
+        if ($module instanceof ContentContainerModule && method_exists($module, 'getContainerPermissions')) {
+            $containerPermissions = $module->getContainerPermissions(new $this->contentcontainer_class);
+            if (!empty($containerPermissions)) {
+                // Don't try to find container permissions in the ContentContainerModule::getPermissions() below
+                // since they are already defined in more proper method ContentContainerModule::getContainerPermissions()
+                return $containerPermissions;
```

Comment:
A ContentContainerModule can also export non container related permissions in `getPermissions()` think those won't be included in this logic?

Reply:
Also note that some container modules simply overwritten the `getPermissions()` instead of using the `getContainerPermissions()`

Reply:
Look good for me. Modules can either export permissions via `getContainerPermissions()` (preferred) or `getPremissions()`.

Reply:
Ok, this permission manager is solely used to determine default permissions in the admin settings.